runtime/shared_common/ROMClassManagerImpl.hpp

```diff
@@ -104,19 +103,13 @@ class SH_ROMClassManagerImpl : public SH_ROMClassManager
 	 *
 	 * Local hashtable (_rcTable) has an entry for each known ROMClass name.
 	 * Key is the fully-qualified string name of the ROMClass
-	 * Value is a RcLinkedListImpl list of ROMClasses stored under that name
-	 * A RcLinkedListImpl is a reference to a ROMClass in the cache.
+	 * Value is a HashLinkedListImpl list of ROMClasses stored under that name
```

Comment:
This whole comment seems strangely placed.

Reply:
This is a comment for  _linkedListImplPool. I have put it before _linkedListImplPool.